docs/cli-options.rst

```diff
@@ -146,6 +146,23 @@ Which Posts to Download
    This flag is recommended when you use Instaloader to update your personal
    Instagram archive.
 
+.. option:: --latest-stamps [STAMPSFILE]
+
+   Works similarly to :option:`--fast-update`, but instead of relying on already
+   downloaded media, the time each profile was downloaded is stored, and only
+   media newer poster after the last download is fetched. This allows updating
```

Comment:
"only media newer poster after the last download is fetched" does not sound right to me (grammatically).